.github/workflows/build-book.yaml

```diff
@@ -80,9 +86,10 @@ jobs:
         uses: dawidd6/action-download-artifact@v3.1.4
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          workflow: ${{ inputs.trigger_workflow }}
+          workflow: ${{ inputs.workflow }}
```

Comment:
I think we can remove this input here since we're relying on `run_id`, right?

Reply:
It was hard-coded as trigger-book-build.yaml, so i don't want to break any thing that needs to input trigger-book-build.yaml

Reply:
Right, but for this step isn't it redundant since always specify `run_id`?

Reply:
I just know that before I started messing with it - it always had run_id and the specified workflow

Reply:
So I don't want to remove it unless it turns out I have to